docs/workspaces.md

```diff
@@ -601,6 +597,26 @@ the storage solution that you are using.
 * `ReadWriteMany` is the least commonly available Access Mode. If you use this access mode and these volumes are available
   to all Nodes within your cluster, you may want to disable the Affinity Assistant.
 
+### Persistent Volume Availability Zone
+
+`Persistent Volumes` are usually "zonal" (i.e. they live within a single Availability Zone and cannot be accessed from a `pod` living in another Availability Zone). When using a workspace backed by a `PersistentVolumeClaim` (typically only available within a Data Center), the `TaskRun`
+`pods` can be scheduled to any Availability Zone in a regional cluster. This leads to two kinds of potential Availability Zone scheduling conflict.
```

Comment:
Can you link to an issue here?

Reply:
I'd still like to see an issue even with the updated docs!

Reply:
issue links updated!